amd_openvx_extensions/amd_rpp/source/tensor/Resample.cpp

```diff
@@ -0,0 +1,298 @@
+/*
+Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+#include "vx_ext_amd.h"
+#define NUM_OF_DIMS 5
+
+struct ResampleLocalData {
+    vxRppHandle *handle;
+    Rpp32u deviceType;
+    Rpp32u nbatchSize;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    float quality;
+    uint maxDstWidth;
+    RpptDescPtr src_desc_ptr;
+    RpptDesc srcDesc;
+    RpptDesc dstDesc;
+    RpptDescPtr dst_desc_ptr;
+    Rpp32s *sampleArray;
+    Rpp32s *sampleChannels;
+    Rpp32f *inRateTensor;
+    Rpp32f *outRateTensor;
+    void *roi_tensor_ptr_src;
+    RpptROI *roi_ptr_src;
+    void *roi_tensor_ptr_dst;
+    RpptROI *roi_ptr_dst;
+    size_t in_tensor_dims[NUM_OF_DIMS];
+    size_t out_tensor_dims[NUM_OF_DIMS];
+    vx_enum in_tensor_type;
+    vx_enum out_tensor_type;
+#if ENABLE_OPENCL
+    cl_mem cl_pSrc;
+    cl_mem cl_pDst;
+#elif ENABLE_HIP
+    void *hip_pSrc;
+    void *hip_pDst;
```

Comment:
I think we don't require HIP and OpenCL related variables in new changes done in all openvx nodes
I am not able to see this variables in brightness.cpp
https://github.com/swetha097/MIVisionX/blob/swbs_m5/audio/pr8/amd_openvx_extensions/amd_rpp/source/tensor/Brightness.cpp

Please check and remove this